typings/externs.d.ts

```diff
@@ -77,29 +89,36 @@ declare global {
       onlyAudits?: string[] | null;
       onlyCategories?: string[] | null;
       skipAudits?: string[] | null;
+      extraHeaders?: Crdp.Network.Headers | null; // See extraHeaders TODO in bin.js
     }
 
     export interface Flags extends SharedFlagsSettings {
-      // Used by both core/ and cli/
-      port: number;
-      hostname: string;
-      output: any;
-      logLevel: 'silent'|'error'|'info'|'verbose';
+      port?: number;
+      hostname?: string;
+      logLevel?: 'silent'|'error'|'info'|'verbose';
+      configPath?: string;
+    }
 
-      // Just used by cli/
+    /**
+     * Flags accepted by Lighthouse, plus additional flags just
+     * for controlling the CLI.
+     */
+    export interface CliFlags extends Flags {
       _: string[];
       chromeFlags: string;
       outputPath: string;
       saveAssets: boolean;
       view: boolean;
-      enableErrorReporting: boolean;
+      enableErrorReporting?: boolean;
```

Comment:
make this a yargs boolean instead. seems weird to have this one optional but the rest not.

Reply:
done

Reply:
wait noooooo, we gotta revert this. it's purposefully not a yargs boolean so that the setting saved to their config will be respected. explicitly setting as false via yargs will always turn error reporting off

Reply:
whoops, very good point. There *was* a reason it wasn't in `yargs.boolean()` already :) I'll revert and add a comment so we don't do that again